### PR TITLE
starting detecting using mediapipe

### PR DESCRIPTION
**Pull request by olaib**
starting detecting poses on images using mediapipe

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    return result
+
+def draw_pose_on_frame(frame, detected_frame,  drawing):
+    if detected_frame.pose_landmarks:
+        DRAWING.draw_landmarks(
+            image=frame,
+            landmark_list=detected_frame.pose_landmarks,
+            connections=mp.solutions.pose.POSE_CONNECTIONS,
+            landmark_drawing_spec=DRAWING.DrawingSpec(
+                color=(250, 250, 250),
```
Comment (gavrielk) [inline main.py]: Assign constants to descriptive variables to enhance readability and simplify future changes ^__^ . Directly using them can lead to confusion and maintenance challenges.

File: main.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                thickness=7,
+                circle_radius=4
+            ),
+            connection_drawing_spec=DRAWING.DrawingSpec(
+                color=(0, 0, 255),
```
Comment (gavrielk) [inline main.py]: Assign constants to descriptive variables to enhance readability and simplify future changes ^__^ . Directly using them can lead to confusion and maintenance challenges.

File: main.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+    while cap.isOpened():
+        ret,frame = cap.read()
+        if not ret:
+            break
+        # frame_to_RGB = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
+        result = detect_pose(frame,POSE_INIT)
+        draw_pose_on_frame(frame,result,DRAWING)
+        cv2.imshow('Video After detecting the pose', frame)
+        if cv2.waitKey(1) & 0xFF == ord('q'):
```
Comment (gavrielk) [inline main.py]: Assign constants to descriptive variables to enhance readability and simplify future changes ^__^ . Directly using them can lead to confusion and maintenance challenges.

File: main.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            break
+    cap.release()
+    cv2.destroyAllWindows()
+
+def main(argv=None) -> None:
```
Comment (gavrielk) [inline main.py:80]: Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be

File: main.py [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+            )
+        )
+        print(detected_frame.pose_landmarks)
+
+def detect_human_pose_from_video(video_path: str):
+    pose = POSE_INIT
+    cap = cv2.VideoCapture(video_path)
+
+    while cap.isOpened():
+        ret,frame = cap.read()
```
Comment (gavrielk) [inline main.py]: How many frames per second do you manage to support?

Comment (olaib) [reply main.py]: maybe 30, I tried to limit the fps to maximum 30 using cv2 see cap.set(cv2.CAP_PROP_FPS, FPS).
I will think about more efficient way to get exactly 30

Comment (excellenteam-scaleup) [reply main.py]: Great, also measure it in the raspberryPi because the performance will be very different than what you get on your PC

**Pull request merged**